src/360-core/browzine-360-core-adapter.js

```diff
@@ -6,31 +6,30 @@ browzine.serSol360Core = (function() {
     return url.replace("api.thirdiron.com", "public-api.thirdiron.com");
   };
 
-  function getQueryVariable(url, key) {
-    var query = url.split("?")[1];
-    var parameters = query.split("&");
-    var value = "";
-
-    for(var i = 0; i < parameters.length; i++) {
-      var pair = parameters[i].split("=");
-
-      if(pair[0] == key) {
-        value = pair[1];
-        return value;
-      }
-    }
+  function getIssn(title) {
+    var issn = "", eissn = "";
 
-    return value;
-  };
+    if(title.identifiers) {
+      var issnIdentifier = title.identifiers.filter(function(identifier) {
+        if(identifier.type && identifier.value) {
+          return identifier.type.toLowerCase() === "issn" && identifier.value.length > 0;
+        }
+      }).pop();
```

Comment:
If we find an ISSN here, can we safely short circuit the rest of this method and return it?

Reply:
Good question Keegan. Doesn't look like there is a built-in ability to break in `filter`. Should be 1 to 2 objects in that `identifiers` array max. Should be fine.

Reply:
About to push an update, think we may be able to short circuit it.

Reply:
Nice!